packages/rancher-istio/charts/configs/istio-base.yaml

```
@@ -73,18 +73,26 @@ spec:
       coreDNSPluginImage: {{ template "system_default_registry" . }}{{ .Values.istiocoredns.pluginImage.repository }}:{{ .Values.istiocoredns.pluginImage.tag }}
       coreDNSTag: {{ .Values.istiocoredns.image.tag }}
     mixer:
-      {{- if .Values.policy.enabled }}
+      {{- if .Values.mixer.policy.enabled }}
       policy:
-        image: {{ template "system_default_registry" . }}{{ .Values.policy.repository}}:{{ .Values.policy.tag }}
+        image: {{ template "system_default_registry" . }}{{ .Values.mixer.policy.repository}}:{{ .Values.mixer.policy.tag }}
       {{- end }}
-      {{- if .Values.telemetry.enabled }}
+      {{- if .Values.mixer.telemetry.enabled }}
       telemetry:
-        image: {{ template "system_default_registry" . }}{{ .Values.telemetry.repository}}:{{ .Values.telemetry.tag }}
+        image: {{ template "system_default_registry" . }}{{ .Values.mixer.telemetry.repository}}:{{ .Values.mixer.telemetry.tag }}
       {{- end }}
     {{- if .Values.pilot.enabled }}
     pilot:
       image: {{ template "system_default_registry" . }}{{ .Values.pilot.repository}}:{{ .Values.pilot.tag }}
     {{- end }}
+    {{- if .Values.telemetry.enabled }}
+    telemetry:
+      enabled: {{ .Values.telemetry.enabled }}
+      v1:
+        enabled: {{ .Values.telemetry.v1.enabled }}
+      v2:
+        enabled: {{ .Values.telemetry.v2.enabled }}
+    {{- end}}
```

I'm not a huge fan of the idea that you need to specify enabled twice. Can we modify this to:
```suggestion
    {{- if or .Values.telemetry.v1.enabled .Values.telemetry.v2.enabled }}
    telemetry:
      enabled: true # is this necessary anymore?
      v1:
        enabled: {{ .Values.telemetry.v1.enabled }}
      v2:
        enabled: {{ .Values.telemetry.v2.enabled }}
    {{- end}}
```

---

I used this idea, but I also do need to still define the field `telemetry.enabled` so I went ahead and used the same if statement to set it